mkdocs.yml

```diff
@@ -27,7 +27,7 @@ extra_javascript:
 copyright: <a target="_blank" href="https://mat3ra.com">Exabyte Inc</a>. All rights reserved. | <a target="_blank" href="https://platform.mat3ra.com">Back to platform</a>
 
 extra:
-    version: "2024.3.14"
+    version: "2024.5.16"
```

Comment:
Remember to update the latest platform version before merging.

Reply:
The latest version is 2024.8.22